modules/@apostrophecms/login/index.js

```diff
@@ -140,7 +140,7 @@ module.exports = {
         },
         async logout(req) {
           if (!req.user) {
-            throw self.apos.error('forbidden', 'You were not logged in.');
+            throw self.apos.error('forbidden', req.t('logoutNotLoggedIn'));
```

Comment:
I saw the bit about using the default locale if no namespace. These strings are in the same localization file as the other strings, so I'm still unclear what the flow is here.

Reply:
Cleaned these up. I forgot to include the namespace. It is required to get the desired result. The default namespace is the default for the entire project, it is not a per module thing.

Reply:
Thus the default namespace is only useful for project level work, which is fine because it gives the convenience to the project developer where it belongs.